File: linearSearch.py
pos=-1
def search(list,n):
    # for i in range (len(list)):
    #     if list[i]== n:
    #         global pos
    #         pos =i
    #         return True
    #     i=i+1
    # return False
    i=0
  
    while i < len(list):
        if list[i]== n:
            global pos
            pos =i
            return True
        i=i+1
    return False


list = [5,10,15,20]
n=5
if search(list,n):
    print(f"{n} is at index {pos} in here this list")
else:
    print(f"{n}is at index not found in given list")

File: LinearSearch.java
import java.util.*;
class LSArray{
   public static int find (int [] arr,int n){
    for (int i=0;i<arr.length;i++){
        if(arr[i]==n)
        {
            return i;
        }
    }
    return -1;
    
   }
}


public class LinearSearch {
    public static void main(String[] args) {
    
    Scanner sc =new Scanner(System.in);
    System.out.println("Enter array size : ");
    int size = sc.nextInt();
    int [] arr =new int[size];
     System.out.println("Enter array Value's : ");
     for(int i =0 ;i< size;i++){
        arr[i]=sc.nextInt();
     }
     System.out.println("Enter array Value's : ");
      int n = sc.nextInt();
      int index = LSArray.find(arr,n);
      if(index !=-1){
        System.out.println( n +" is find the index in the given list in position "+ index);
      }
      else{
                System.out.println( "Sorry Bro this " + n +" not found that this list ");

      }

    }
}